client/.eslintrc.js

```diff
@@ -14,10 +15,8 @@ module.exports = {
 	},
 	overrides: [
 		{
-			files: [ '**/test/**/*' ],
-			rules: {
-				'import/no-nodejs-modules': 'off',
-			},
+			files: [ './webpack.*.js', './**/docs/example.jsx', './server/**/*', '**/test/**/*' ],
```

Comment:
Should JSX files get the node config? I guess the broader question is: should we restrict use of the node config to code which runs in node?

Reply:
Good point. This is a mistake, the only thing `'./**/docs/example.jsx'` should get is `no-console`, as we use it a lot in the examples. But there is no reason to apply other node-only rules.